addons/hearing/functions/fnc_updateVolume.sqf

```diff
@@ -16,6 +16,12 @@
  */
 #include "script_component.hpp"
 
+if (!alive ACE_player) then {
```

Comment:
Can this not be `exitWith` ? If `ACE_player` is not alive, is anything that comes after it still relevant?